src/app/modules/shared/components/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Output, Component, EventEmitter, Input } from '@angular/core';
+
+
+@Component({
+  selector: 'app-dropdown',
+  templateUrl: './dropdown.component.html',
+  styleUrls: ['./dropdown.component.scss']
+})
+export class DropdownComponent {
+
+  @Input() info: any;
+  @Output() updateInfo: EventEmitter<any> = new EventEmitter();
+
+  dataChange: any = {
+    id: '',
+    name: '',
+    description: '',
+    tenant_id: '',
+    status: '',
+    key: '',
+    _status: false,
+    btnColor: '',
+    btnIcon: '',
+    btnIconTwo: '',
+    btnName: '',
+    iconColor: ''
+  };
+
+  changePropertiesItem(item: any){
```

Comment:
Maybe you can use object destructuring, like: 
```suggestion
  changePropertiesItem({ btnName, btnIcon} ){
```
Only if you consider necessary you can apply this change.

Reply:
done